docs/interfaces/rest.md

```diff
@@ -1,70 +0,0 @@
-<!--
```

Comment:
removed docs concerning legacy REST. We already have:
- docs for grpc-gateway
- migration from REST